PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalInternalClientUnitTest.kt

```diff
@@ -0,0 +1,1229 @@
+                any()
+            )
+        }
+        val result = slot.captured
+        val actual = JSONObject(result)
+        Assert.assertFalse(actual.has("line_items"))
+    }
+    @Test
+    fun sendRequest_whenRiskCorrelationIdNotNull_setsClientMetadataIdToRiskCorrelationId() {
+        every {
+            dataCollector.getClientMetadataId(
+                eq(context),
+                eq(configuration),
+                eq(true)
+            )
+        } returns "sample-client-metadata-id"
+        val braintreeClient = MockkBraintreeClientBuilder()
+            .sendPOSTSuccessfulResponse(Fixtures.PAYPAL_HERMES_RESPONSE)
+            .build()
+        every { merchantRepository.authorization } returns clientToken
+
+        val sut = PayPalInternalClient(
+            braintreeClient,
+            dataCollector,
+            apiClient,
+            deviceInspector,
+            merchantRepository,
+            getReturnLinkUseCase,
+            setAppSwitchUseCase,
+            getAppSwitchUseCase,
+            analyticsParamRepository
+        )
+
+        val payPalRequest = PayPalCheckoutRequest("1.00", true)
+        payPalRequest.riskCorrelationId = "risk-correlation-id"
+
+        sut.sendRequest(context, payPalRequest, configuration, payPalInternalClientCallback)
+
+        val slot = slot<PayPalPaymentAuthRequestParams>()
+        verify {
+            payPalInternalClientCallback.onResult(capture(slot), null)
+        }
+        val payPalPaymentAuthRequestParams = slot.captured
+        Assert.assertEquals("risk-correlation-id", payPalPaymentAuthRequestParams.clientMetadataId)
+    }
+
+    @Test
+    fun sendRequest_whenRiskCorrelationIdNull_setsClientMetadataIdFromPayPalDataCollector() {
+        every {
+            dataCollector.getClientMetadataId(
+                eq(context),
+                any(),
+                eq(configuration)
+            )
+        } returns "sample-client-metadata-id"
+        val braintreeClient = MockkBraintreeClientBuilder()
+            .sendPOSTSuccessfulResponse(Fixtures.PAYPAL_HERMES_RESPONSE)
+            .build()
+
+        every { merchantRepository.authorization } returns clientToken
+        val sut = PayPalInternalClient(
+            braintreeClient,
+            dataCollector,
+            apiClient,
+            deviceInspector,
+            merchantRepository,
+            getReturnLinkUseCase,
+            setAppSwitchUseCase,
+            getAppSwitchUseCase,
+            analyticsParamRepository
+        )
+
+        val payPalRequest = PayPalCheckoutRequest("1.00", true)
+
+        sut.sendRequest(context, payPalRequest, configuration, payPalInternalClientCallback)
+
+        val slot = slot<PayPalPaymentAuthRequestParams>()
+        verify {
+            payPalInternalClientCallback.onResult(capture(slot), null)
+        }
+        val payPalPaymentAuthRequestParams = slot.captured
+        TestCase.assertNull(payPalRequest.riskCorrelationId)
```

Comment:
Instead of using `TestCase.assertNull` can we stay consistent and use `Assert.assertNull`?